dojo/tools/cargo_audit/parser.py

```diff
@@ -0,0 +1,58 @@
+import json
+import hashlib
+from dojo.models import Finding
+
+
+class CargoAuditParser(object):
+    """
+    A class that can be used to parse the cargo audit JSON report file
+    """
+
+    def get_scan_types(self):
+        return ["CargoAudit Scan"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "CargoAudit Scan"
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Import JSON output for cargo audit scan report."
+
+    def get_findings(self, filename, test):
+        data = json.load(filename)
+        dupes = {}
+        if data.get('vulnerabilities'):
+            for item in data.get('vulnerabilities').get('list'):
+                advisory = item.get('advisory')
+                vuln_id = advisory.get('id')
+                title = advisory.get('title')
+                description = advisory.get('description')
+                date = advisory.get('date')
+                cve = advisory.get('aliases')[0]
+                url = advisory.get('url')
+                package_name = item.get('package').get('name')
+                package_version = item.get('package').get('version')
+                severity = "High"
+
+                dupe_key = hashlib.sha256(
+                    (vuln_id + cve + date + package_name + package_version).encode('utf-8')
+                ).hexdigest()
+
+                if dupe_key in dupes:
+                    finding = dupes[dupe_key]
+                    finding.nb_occurences += 1
+                else:
+                    finding = Finding(
+                        title=title,
+                        test=test,
+                        severity=severity,
+                        cve=cve,
+                        url=url,
```

Comment:
```suggestion
```
You can remove this attribute. This attribute is not used in the UI, it could be removed in the future.